plugins/cpp_metrics/model/include/model/cppastnodemetrics.h

```diff
@@ -0,0 +1,36 @@
+#ifndef CC_MODEL_CPPASTNODEMETRICS_H
+#define CC_MODEL_CPPASTNODEMETRICS_H
+
+//#include <../../../../plugins/cpp/model/include/model/cppastnode.h>
```

Comment:
Commented out code should be removed.

Reply:
Fixed in 667361582fcf5b58c66ed660fe37a2aa57b5fa59.